doc/man7/EVP_PKEY-EC.pod

```diff
@@ -79,6 +79,10 @@ Enable Cofactor DH (ECC CDH) if this value is 1, otherwise it uses normal EC DH
 if the value is zero. The cofactor variant multiplies the shared secret by the
 EC curve's cofactor (note for some curves the cofactor is 1).
 
+See also L<EVP_KEYEXCH-ECDH(7)> for the related
+B<OSSL_EXCHANGE_PARAM_EC_ECDH_COFACTOR_MODE> parameter that can be set on a
+per-operation basis.
```

Comment:
Not sure why there is a hyphen here, I'd have used a space.
Still, it's pre-existing so no need to change it.

Reply:
At least in US english, hyphenating compound adjectives is the standard way to do things.  It's not a (per (operation basis)), it's a ((per-operation) basis).